### PR TITLE
1182b multi ia config

### PR DESCRIPTION
**Pull request by drewblount**
Updates ia config class so that, rather than taking a single config file on creation, it combines every config file (matching IA.*.json) in the ia-configs directory. Reorganizes directory so that only seals and zebras are included, rest of configs now live in a subdir.

Most of the change here is in the IaConfig class init method.

I am definitely open to input re: adding a _config file validation_ step during the init process. It sounds nice in theory; here's why I didn't do it:

- the tests have good coverage and will catch if we break any existing behavior by adding new files
- we'd want to write tests that new configs are parsed correctly anyway
- We don't want to force any file to include values like _detectors or _identifiers bc they could simply reuse the ones defined in other configs. Eg we might add an IA.okapi.json file that points at zebra ID and detector algos and only has an entry for the okapi taxonomy.

So file validation is done in the tests rather than the parser itself, except the file does have to pass json.loads in the code.

### CODE REVIEW

File: app/modules/ia_config_reader.py [hunk 1]
```diff
@@ -194,6 +211,6 @@ def get_detect_model_frontend_data(self):
         return result
 
 
-ic = IaConfig()
+# ic = IaConfig()
```
Comment (hwindsor) [inline app/modules/ia_config_reader.py:214]: and in the constructor, if the code is finished with, it should be removed, not commented out

Comment (drewblount) [reply app/modules/ia_config_reader.py]: Removed

**Review by hwindsor — APPROVED** [review]
LGTM

**Review by naknomum — APPROVED** [review]
lgtm

**Pull request merged**